packages/platform-core/src/errors/amplify_error.ts

```diff
@@ -129,6 +132,17 @@ export abstract class AmplifyError<T extends string = string> extends Error {
         error
       );
     }
+    if (error instanceof Error && isENotFoundError(error)) {
+      return new AmplifyUserError(
+        'DomainNotFoundError',
+        {
+          message: 'Unable to establish a connection to a domain',
+          resolution:
+            'Ensure domain name is correct and network connection is stable.',
+        },
```

Comment:
Is there any way to extract information from error about which domain was attempted?

Reply:
The original error that is passed on line 143 has the domain that was attempted, the complete error message would be something like `getaddrinfo ENOTFOUND *.amazonaws.com`